src/org/javarosa/core/model/instance/TreeElement.java

```diff
@@ -438,6 +437,11 @@ public TreeElement getBindAttribute(String namespace, String name) {
     /**
      * get value of the bind attribute with namespace:name' in the list
      *
+     * Example
+     *   Accessing a bind simply called "assertion" (has no namespace); can be called as follows:
+     *   <code>getBindAttributeValue(null, "assertion");</code>
+     * @param namespace a namespace string if provided, else null
+     * @param name the name of an attribute of a <code><bind/></code> node, e.g. "myAttr" if the bind node were <code><bind myAttr="myVal"/></code>
      * @return String
```

Comment:
For me when I was using JavaRosa as a library in XFormTest, this actually would have been helpful to me. For me, `@return String` and the line on `@param namespace` are redundant. But if I already have a Javadoc with a leading paragraph (and maybe also the `@param name`, I think it would probably be bad styling to leave out the other params / returns?

If you have a specific suggestion for how to rephrase, could you write it here and I will update with a commit?

Reply:
Never mind, I see you did that in the next comment.